samples/I2CBMP280.js

```diff
@@ -0,0 +1,244 @@
+    GLCD_CMD_CURSOR_RETURN : 1 << 1,
+    GLCD_CMD_INPUT_SET : 1 << 2,
+    GLCD_CMD_DISPLAY_SWITCH : 1 << 3,
+    GLCD_CMD_CURSOR_SHIFT : 1 << 4,
+    GLCD_CMD_FUNCTION_SET : 1 << 5,
+    GLCD_CMD_SET_CGRAM_ADDR : 1 << 6,
+    GLCD_CMD_SET_DDRAM_ADDR : 1 << 7,
+// Defines for the LCD_FUNCTION_SET
+    GLCD_FS_8BIT_MODE : 1 << 4,
+    GLCD_FS_ROWS_2 : 1 << 3,
+    GLCD_FS_ROWS_1 : 0 << 3,
+    GLCD_FS_DOT_SIZE_BIG : 1 << 2,
+    GLCD_FS_DOT_SIZE_LITTLE : 0 << 2,
+// Defines for the GLCD_CMD_DISPLAY_SWITCH options
+    GLCD_DS_DISPLAY_ON : 1 << 2,
+    GLCD_DS_DISPLAY_OFF : 0 << 2,
+    GLCD_DS_CURSOR_ON : 1 << 1,
+    GLCD_DS_CURSOR_OFF : 0 << 1,
+    GLCD_DS_BLINK_ON : 1 << 0,
+    GLCD_DS_BLINK_OFF : 0 << 0,
+    REGISTER_POWER : 0x08,
+    REGISTER_R : 0x04,
+    REGISTER_G : 0x03,
+    REGISTER_B : 0x02
+}
+
+/* Globals */
+var i2cDevice = i2c.open({ bus: 0, speed: 100 });
+var reader;
+var dig_T1;
+var dig_T2;
+var dig_T3;
+var setupData = new Buffer(2);
+var msgData = new Buffer("@Temperature is ");
+var tempData = new Buffer(" ##.#C");
+var changeData = new Buffer(" +");
+var baseTemp = prevTemp = 25;
+var firstRead = true;
+var red = green = blue = 200;
+
+function uint16toInt16(value) {
+    if ((value & 0x8000) > 0) {
+        value = value - 0x10000;
+    }
+    return value;
+}
+
+function clear() {
+    var clearData = new Buffer([0, glcd.GLCD_CMD_SCREEN_CLEAR]);
+    i2cDevice.write(glcd.GROVE_LCD_DISPLAY_ADDR, clearData);
+}
+
+function resetCursor(col, row) {
+    // Reset the cursor position
+    row = row === undefined ? 0 : row;
+    col = col === undefined ? 0 : col;
+
+    if (row == 0) {
+        col |= 0x80;
+    } else {
+        col |= 0xC0;
+    }
+    
+    setupData.writeUInt8(glcd.GLCD_CMD_SET_DDRAM_ADDR, 0);
+    setupData.writeUInt8(col, 1);
+    i2cDevice.write(glcd.GROVE_LCD_DISPLAY_ADDR, setupData);
+}
+
+function changeRGB(red, green, blue) {
+    // Valid range for color is 0 - 255    
+    var redData = new Buffer([glcd.REGISTER_R, red]);
+    var greenData = new Buffer([glcd.REGISTER_G, green]);
+    var blueData = new Buffer([glcd.REGISTER_B, blue]);
+
+    // Send messages as close together as possible,
+    // so that the color change is smoother
+    i2cDevice.write(glcd.GROVE_RGB_BACKLIGHT_ADDR, blueData);
+    i2cDevice.write(glcd.GROVE_RGB_BACKLIGHT_ADDR, redData);
+    i2cDevice.write(glcd.GROVE_RGB_BACKLIGHT_ADDR, greenData);
+}
+
+function updateColor(newTemp) {
+    var diff = newTemp - baseTemp;
+    var change = diff > 0 ? diff * 40 : diff * -40;
+    change = change > 255 ? 255 : change | 0;
+
+    if (baseTemp < newTemp) {
+        blue = 200 - change < 0 ? 0 : 200 - change;
+        green = blue;
+        red = 200 + change > 255 ? 255 : 200 + change;
+    } else {
+        red = 200 - change < 0 ? 0 : 200 - change;
+        green = red;
+        blue = 200 + change > 255 ? 255 : 200 + change;
+    }
+
+    prevTemp = newTemp;
+    changeRGB(red | 0, green | 0, blue | 0);
```

Comment:
i don't think you'll need to do the | 0 here to convert it to int, since change is already an integer.